### ISSUE TITLE
Finite Mass

### ISSUE THREAD

**Issue by ghost** [idmillington/cyclone-physics#23]
In the book (p 47) it says the inverse mass is used for representing infinite mass 

> > .. objects of infinite mass have a zero inverse mass

but the code in particle.cpp reads

```
bool Particle::hasFiniteMass() const
{
    return inverseMass >= 0.0f;
}
```

causing the bug

`setInverseMass(0); hasFiniteMass() == true`

**Comment by giordi91** [comment]
agreed, probably best would be to check  if inverse mass is  equal to zero (of course using a bias to compensate for numerical precision). I will try to fix this and make a pull request.
